system-addon/common/Reducers.jsm

```diff
@@ -95,6 +99,19 @@ function TopSites(prevState = INITIAL_STATE.TopSites, action) {
   }
 }
 
+function Dialog(prevState = INITIAL_STATE.Dialog, action) {
+  switch (action.type) {
+    case at.DELETE_HISTORY_URL_CONFIRM:
```

Comment:
Isn't this action really `DIALOG_OPEN` since Dialog is a multi-purpose component?